fastfood/food.py

```diff
@@ -141,85 +151,29 @@ def update_cookbook(cookbook, templatepack, stencilset_name, **options):
     return written_files, cookbook
 
 
-def create_new_cookbook(cookbook_name, templatepack,
-                        cookbooks_home, force=False):
+def create_new_cookbook(cookbook_name, cookbooks_home, force=False):
```

Comment:
At the moment, `force` isn't used anymore. We could change this function so that it fails if you try to call `create_new_cookbook` where the target directory either already exists or exists and is non-empty (one or the other). Then, including `force=True` would completely delete the target directory (if it exists) and re-create it no matter what.